source/common/config/utility.cc

```diff
@@ -25,6 +25,8 @@
 namespace Envoy {
 namespace Config {
 
+const std::string Wildcard = "*";
```

Comment:
Please use constexpr absl::string_view instead.

Reply:
I think you should be able to move it to source/common/config/delta_subscription_state.cc too

Reply:
Ahh, I see that it's used in multiple files; not sure it's worth dragging in the whole header file just to get a definition for an '*' though. Maybe just define it where it's used? Not sure what others will think of this.

Reply:
> Please use constexpr absl::string_view instead.

Will try it out.

> Ahh, I see that it's used in multiple files; not sure it's worth dragging in the whole header file just to get a definition for an '*' though. Maybe just define it where it's used? Not sure what others will think of this.

Frankly speaking, I'm now thinking that if `constexpr absl::string_view` worked then I'd put it in `envoy/config/subscription.h`.